builder/virtualbox/ovf/builder.go

```diff
@@ -48,6 +50,9 @@ func (b *Builder) Run(ui packer.Ui, hook packer.Hook, cache packer.Cache) (packe
 	state.Put("hook", hook)
 	state.Put("ui", ui)
 
+	// Get extension from SourcePath
+	extension := filepath.Ext(b.config.SourcePath)
```

Comment:
What would be the different extensions we would expect? I was assuming only .ova

what happens if someone passes an ovf on accident?